src/deprecation.h

```diff
@@ -13,7 +13,7 @@
 // Per https://zips.z.cash/zip-0200
 // Shut down nodes running this version of code, 16 weeks' worth of blocks after the estimated
 // release block height. A warning is shown during the 14 days' worth of blocks prior to shut down.
-static const int APPROX_RELEASE_HEIGHT = 2476976;
+static const int APPROX_RELEASE_HEIGHT = 2514600;
```

Comment:
This reminds me that we need to automate this "subtract 24 from the approx release height" so that the published EoS heights are round numbers (I've been doing this manually for my releases).

Reply:
I will do that for the release.